src/command_modules/azure-cli-devops/azure/cli/command_modules/devops/custom.py

```diff
@@ -0,0 +1,215 @@
+    :param registry_resource_group: Registry esource group name
+    :type registry_resource_group: String
+    :param registry_name: Registry name
+    :type registry_name: String
+    """
+    # Ensure docker-compose file is correct.
+    ensure_docker_compose()
+
+    #Call the RP
+    call_rp_configure_cicd(name, resource_group_name, vsts_account_name, vsts_project_name, registry_name, registry_resource_group, get_valid_remote_url(remote_url, remote_access_token), remote_access_token)
+    return
+
+def get_valid_remote_url(remote_url = None, remote_access_token = None):
+    """
+    Calls the git repo parser to check for non VSTS repository and returns remote url for repository.
+    """
+    remote_url = remote_url or get_remote_url()
+    repo_info = git_repo_parser.get_repo_info(remote_url)
+
+    if repo_info['repo_type'] != 'vsts' and not remote_access_token:
+        raise CLIError("--remote-access-token is required for non VSTS repositories")
+
+    return remote_url
+
+def call_rp_configure_cicd(name, resource_group_name, vsts_account_name, vsts_project_name, registry_name, registry_resource_group, remote_url, remote_access_token, create_release = True):
+    """
+    Calls the RP to build and deploy the service(s) in the cluster.
+
+    :param name: ACS resource name
+    :type name: String
+    :param resource_group_name: Resource group name
+    :type resource_group_name: String
+    :param vsts_account_name: VSTS account name to use/create
+    :type vsts_account_name: String
+    :param vsts_project_name: VSTS project name to create
+    :type vsts_project_name: String
+    """
+    profile = Profile()
+    cred, subscription_id, _ = profile.get_login_credentials()
+    
+    o_auth_token = cred.signed_session().headers['Authorization']
+       
+    data = {
+        'vstsAccountName': vsts_account_name,
+        'vstsProjectName': vsts_project_name,
+        'token': o_auth_token,
+        'registryName': registry_name,
+        'registryResourceGroup': registry_resource_group,
+        'remoteToken': remote_access_token,
+        'remoteUrl': remote_url,
+        'createRelease' : create_release
+    }
+
+    configure_ci_cd_url = RP_URL.format(
+        subscription_id=subscription_id,
+        resource_group_name=resource_group_name,
+        container_service_name=name) + '/configureCI?api-version=' + API_VERSION
+
+    headers = {'Content-type': 'application/json', 'Authorization': o_auth_token}
+    req = requests.post(configure_ci_cd_url, data=json.dumps(data), headers=headers, timeout=600)
+    while req.status_code == 202:  # Long-running operation
+        time.sleep(10)
+        req = requests.get(BASE_URL + req.headers['Location'], headers=headers, timeout=600)
+    json_request = req.json()
+    if req.status_code != 200:
+        raise CLIError('Error: ' + str(req.status_code) + '. Could not configure CI/CD: ' + req.text)
+    print (json.dumps(json_request, indent=4, sort_keys=True))
```

Comment:
Commands shouldn't print output to stdout by themselves.
If this print really is needed, should use the logger.

Also, if for some reason you do need to have this print statement go to stdout, you should add `from __future__ import print_function` to the very top of the file to support both Python 2 & 3